.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: go
 go:
   - 1.9
+  - 1.8
```

Comment:
We should stick to 1.9 unless there's a huge demand for supporting 1.8.

Reply:
I'm not sure about the tradeoff between supporting more versions and losing the 1.9 specific features, but I agree we can punt on this PR and evaluate the tradeoff when demand for 1.8 (or below) really comes.